catalog/admin/admin-poller-service/src/test/java/org/codice/ddf/catalog/admin/poller/AdminPollerTest.java

```diff
@@ -183,11 +183,11 @@ protected AdminSourceHelper getHelper() {
                 // Mock out the metatypes
```

Comment:
You should use the schematron and other values to test more intricate pattern matching.